src/index.js

```diff
@@ -278,9 +278,41 @@ export function genHLines(width, height, options, override) {
 	return(r)
 }
 
+function genOneBlob(rand, size, initRadius, distance, opt) {
+	const tmp = rand() * 2 * Math.PI;
+
+	const initAngle = getRange(opt.numControls).map(x => tmp + x/opt.numControls*2*Math.PI)
+	const center = new Point(size/2, size/2)
+
+	let data = Array(opt.numControls)
+		for (let i = 0; i < opt.numControls; i ++) {
+		data[i] = {point: movePoint(
+												movePoint(center, initAngle[i], initRadius),
+												rand() * Math.PI*2,
+												rand() * opt.posWindowSize)}
+		data[i].angle = rnd(initAngle[i] + Math.PI/2 - opt.angleWindowSize/2, initAngle[i] + Math.PI/2 + opt.angleWindowSize/2)
+		data[i].ctrl = movePoint(data[i].point, data[i].angle, -1*rnd(distance*(1-opt.handleWindowSize), distance*(1+opt.handleWindowSize)))
+		data[i].ctrl_alt = movePoint(data[i].point, data[i].angle, rnd(distance*(1-opt.handleWindowSize), distance*(1+opt.handleWindowSize)))
+	}
+
+	let path = "M " + data[0].point + " "
+		+ "C " + data[0].ctrl_alt + ", " + data[1].ctrl + ", " + data[1].point + " "
+	for (let i = 2; i < opt.numControls; i ++) {
+		path += "S " + data[i].ctrl + ", " + data[i].point + " "
+	}
+	path += "S " + data[0].ctrl + ", " + data[0].point
+
+	return({path: path, data: data})
+}
+
 export function genBlob(size, options) {
+	genHBlobs(size, options)
```

Comment:
Need to return the 0th element since `genHBlobs` now returns an array.
```suggestion
	return genHBlobs(size, options)[0]
```